metricflow/plan_conversion/instance_converters.py

```diff
@@ -803,6 +803,7 @@ def transform(self, instance_set: InstanceSet) -> SelectColumnSet:  # noqa: D102
         )
 
 
+# TODO: delete this class & all uses. It doesn't do anything.
```

Comment:
This class is supposed to change the column names, but if the specs didn't change, then the column names shouldn't either, so it seems like it doesn't do anything. LMK if I'm overlooking something here!